coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -46,27 +56,22 @@ function AclApprovals() {
     AclRequestsForApprover,
     Error
   >({
-    queryKey: ["aclRequests", activePage, environment, status, aclType, topic],
+    queryKey: [
+      "aclRequests",
+      currentPage,
+      currentAclType,
+      currentEnv,
+      currentStatus,
+      currentTopic,
+    ],
     queryFn: () =>
       getAclRequestsForApprover({
-        pageNo: String(activePage),
-        env: environment,
-        requestStatus: status,
-        aclType,
-        topic,
+        pageNo: String(currentPage),
+        env: currentEnv,
+        requestStatus: currentStatus,
+        aclType: currentAclType,
+        topic: currentTopic,
       }),
-    onSuccess: (data) => {
-      queryClient.refetchQueries(["getRequestsWaitingForApproval"]);
-      // If through filtering a user finds themselves on a non existent page, reset page to 1
-      // For example:
-      // - one request returns 4 pages of results
-      // - navigate to page 4
-      // - change filters, to a request that returns 1 page of results
-      // - if not redirected to page 1, table won't be able to handle pagination (clicking "Back" will set page at -1)
-      if (data.entries.length === 0 && activePage !== 1) {
-        handleChangePage(1);
-      }
-    },
```

Comment:
Just to double check that this was not accidentally removed as i did not spot a replacement. Though clicking the ui this seems to work.

Reply:
Indeed, part of the issue that this is resolving is removing this refetching and page setting logic by making the `page` reset to `1` on filtering changes ^^